src/components/RightSidebar/Participants/ParticipantsList/ParticipantsListVirtual.vue

```diff
@@ -0,0 +1,75 @@
+<!--
+  - @copyright Copyright (c) 2023 Maksim Sukharev <antreesy.web@gmail.com>
+  -
+  - @author Grigorii Shartsev <me@shgk.me>
+  -
+  - @license AGPL-3.0-or-later
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -->
+
+<template>
+	<RecycleScroller ref="scroller"
+		item-tag="ul"
```

Comment:
It makes structure:
```
ul
  li
    ul
      li
        participant1
  li
    ul
      li
        participant2
  li
    ul
      li
        participant3    
```

In `LeftSidebar` this structure was the only possible, because `NcListItem` always renders as `li` element and `NcAppSidebar` always renders as `<ul>`. But here we can use any layout, making an entire scroller a `ul` element and each item true `li` element.

Reply:
`<Participant />` is already a `li` element, which is used in flat `ul` lists (ParticipantsList, BreakoutRoomItem)

RecycleScroller creates some additional wrappers by default:
![image](https://github.com/nextcloud/spreed/assets/93392545/6efcb288-6089-4e5f-8f7d-f184826bfa8b)

the best we could do is
```html
<div virtual-scroller>
  <li>
    <ul>
      <Participant>
```

Reply:
> `<Participant />` is already a `li` element,

Yes, but it is our component, and we can easily make it configurable with dynamic root node
```vue
<component :is="tag">
</component>

<script>
export default {
  props: {
    tag: {
      type: String,
      default: 'li',
    }
  }
}
</script>
```

It could also be useful for `BreakoutRoomItem` where the structure is also `ul>li>ul>li>1-item`

Then in RecycleScroller we can use `list-tag="ul" item-tag="li"`

Reply:
Updated, but `BreakoutRoomItem` was fine, it has structure `ul "BR tab" > li "BR item" > ul "participants list" > <Participant tag="li"/>`